packages/thrift-server-core/src/main/binary.ts

```diff
@@ -27,7 +24,7 @@ export function readByte(byte: number): number {
  * @param offset Offset
  */
 export function readI16(bytes: Buffer, offset: number = 0): number {
-    return bytes.readInt16BE(offset, noAssert)
```

Comment:
Buffer read/write functions no longer accept `noAssert` as an argument from > Node 10: https://nodejs.org/docs/latest-v10.x/api/buffer.html#buffer_buf_readint16le_offset.

Reply:
Based on this abstraction it appears so, though there are functions that call out Little Endian explicitly where needed as well: https://github.com/creditkarma/thrift-server/pull/170/files#diff-6b114b0faf6c5f7871f2e3d81b0689e833eccf2c77b204b5db5fe391636cd3adL104